### DISCUSSION TITLE
an issue about:Processing large station networks

### DISCUSSION THREAD

**Discussion by sunyu0703** [groops-devs/groops#304] [General]
Dear Groops Team,
I am writing to report  I encountered while using the 030groopsGnssProcessing.xml for Advanced: Processing large station networks.
When I select four satellite systems, namely GPS, GLONASS, GALILEO, and BEIDOU, an error message "Cannot setup ambiguities. Network is separated into independent parts!?" pops up. However, when I only choose GPS and GALILEO, no such problem occurs.
I have attached my 030groopsGnssProcessing.xml and 030.log files for your reference. I hope you can help me identify and resolve this problem.
Thank you in advance for your attention and assistance.
Best regards,
SunYu
[030_log.txt](https://github.com/user-attachments/files/18270299/030_log.txt)
[030advancedgroops00.zip](https://github.com/user-attachments/files/18270309/030advancedgroops00.zip)

**Comment by tmayerguerr** [comment]
Hi,

At the moment, our ambiguity setup algorithm is not perfect and is not able to handle all possible combinations of observations. It is on our agenda to improve the algorithm.

As you can see in your log, many stations are disabled, e.g. `abpo: L7IC** and L7DC** must be observed together`. At the moment we need a common epoch for different phase observations on the same frequency.

You can try to remove some observations with GnssProcessing->[receiver](https://groops-devs.github.io/groops/html/gnssReceiverGeneratorType.html)->ignoreType (e.g. `L7DC**`) to get the algorithm to work.

Best regards
Torsten

**Reply by sunyu0703** [reply]
Can I find relevant information from this paper on "On vector integer bootstrapping implementations in the estimation of satellite orbits and clocks based on small global networks", doi. org/10.1016/j.asr. 2021.09.023
